docs/setup/administrators/configuration.md

```diff
@@ -141,6 +141,12 @@ Possible values are: fund, round, status, lead, reviewers, screening_statuses, c
 
     SUBMISSIONS_TABLE_EXCLUDED_FIELDS = env.list('SUBMISSIONS_TABLE_EXCLUDED_FIELDS', [])
 
+### Score diplayed on reviews page.
+
+If True sum of scores will be displayed otherwise average of scores will be displayed
+
+    DISPLAY_TOTAL_REVIEW_SCORE = env.bool('DISPLAY_TOTAL_REVIEW_SCORE', default = False)
```

Comment:
Make it an option instead of boolean. Current boolean make a lot of implicit assumption. It's a either/or situation but not reflected well with the name of the variable. 

Make this setting a take a value, and have it also accept empty for use-cases where no final summary is desired by any setup. 

Suggestion:
```
from typing import Literal

REVIEWS_FINAL_SCORE_METHOD: Literal['sum', 'avg', ''] = env.str('REVIEWS_SUMMARY_METHOD', default='sum')
```